[5.10] Adding permission for resend-code endpoint at toml level (#4769)
wso2/docs-is: en/identity-server/5.10.0/docs/develop/using-the-self-sign-up-rest-apis.md

!!! info "Related Links" 
    For information on self-registration via the UI instead, see [Self-Registration and Account Confirmation](../../learn/self-registration-and-account-confirmation).

## Enhance default permissions for the resend-code endpoint.

> The [resend-code endpoint](https://api-docs.wso2.com/apidocs/is/is510/self-registration/#!/operations#SelfRegister#resendCodePost) of the self sign-up rest APIs is used to resend the confirmation code to an authenticated user. While no scopes are required to invoke this API by default, we recommend limiting permissions to this endpoint using scopes, prior to production deployment.

Reviewer: ```suggestion
The [resend-code endpoint](https://api-docs.wso2.com/apidocs/is/is510/self-registration/#!/operations#SelfRegister#resendCodePost) of the self sign-up rest APIs is used to resend the confirmation code to an authenticated user. While no scopes are required to invoke this API by default, we recommend restricting access to this endpoint using scopes, before deploying to production.
```

> To do so, add the following configurations to the `deployment.toml` file

Reviewer: ```suggestion
To do so, add the following configurations to the `<IS_HOME>/repository/conf/deployment.toml/deployment.toml` file
```

> To do so, add the following configurations to the `<IS_HOME>/repository/conf/deployment.toml/deployment.toml` file

Reviewer: ```suggestion
To do so, add the following configurations to the `<IS_HOME>/repository/conf/deployment.toml` file
```
